hudi-common/src/test/java/org/apache/hudi/common/testutils/reader/HoodieFileSliceTestUtils.java

```diff
@@ -247,36 +245,31 @@ public static HoodieBaseFile createBaseFile(
       Schema schema,
       String baseInstantTime
   ) throws IOException {
-    Configuration hadoopConf = new Configuration();
+    StorageConfiguration<Configuration> conf = HoodieTestUtils.getDefaultStorageConfWithDefaults();
 
     // TODO: Optimize these hard-coded parameters for test purpose. (HUDI-7214)
-    BloomFilter filter = BloomFilterFactory.createBloomFilter(
-        1000,
-        0.0001,
-        10000,
-        BloomFilterTypeCode.DYNAMIC_V0.name());
-    HoodieAvroWriteSupport<IndexedRecord> writeSupport = new HoodieAvroWriteSupport<>(
-        new AvroSchemaConverter().convert(schema),
-        schema,
-        Option.of(filter),
-        new Properties());
-    HoodieParquetConfig<HoodieAvroWriteSupport> parquetConfig = new HoodieParquetConfig(
-        writeSupport,
-        CompressionCodecName.GZIP,
-        ParquetWriter.DEFAULT_BLOCK_SIZE,
-        ParquetWriter.DEFAULT_PAGE_SIZE,
-        1024 * 1024 * 1024,
-        hadoopConf,
-        0.1,
-        true);
-
-    try (HoodieAvroFileWriter writer = (HoodieAvroFileWriter) ReflectionUtils.loadClass(HOODIE_AVRO_PARQUET_WRITER,
-        new Class<?>[] {StoragePath.class, HoodieParquetConfig.class, String.class, TaskContextSupplier.class, boolean.class},
-        new StoragePath(baseFilePath),
-        parquetConfig,
-        baseInstantTime,
-        new LocalTaskContextSupplier(),
-        true)) {
+    HoodieConfig cfg = new HoodieConfig();
```

Comment:
Should this test class be moved to `hudi-hadoop-common` module? `HoodieAvroWriteSupport<IndexedRecord>` needs to be tested.

Reply:
not sure what you are suggesting here

Reply:
Basically, making sure that the bloom filter writing is tested with `HoodieAvroWriteSupport`.

Reply:
<img width="1329" alt="image" src="https://github.com/apache/hudi/assets/26940621/00c55107-d729-4ced-bacc-6820ca5e8c67">
Ran a test with the debugger and this shows that bloom filter is used